cmd/virt-handler/Dockerfile

```diff
@@ -28,4 +28,9 @@ RUN yum -y install wget && \
 
 COPY virt-handler /virt-handler
 
+RUN groupadd -g 993 libvirt
```

Comment:
Relying on matching UIDs across different containers is not very attractive IMHO and does little to nothing for security.